source-browser.html

```diff
@@ -67,21 +91,21 @@ <h1 id="loading" class="m1 center display-none">⌛</h1>
         CACHE_KEY = `${ORGANIZATION}-cache`,
         EXPIRY_KEY = `${ORGANIZATION}-expiry`,
         EXPIRY_MILLIS = 60000,
-        TOPICS = ['languages', 'trunk', 'staging', 'nursery', 'incubator', 'tools'].map(topic => ({
+        LIST = ['languages', 'trunk', 'staging', 'nursery', 'incubator', 'tools'],
```

Comment:
Please don't use entirely unhelpful variables names like `list`. In fact, this change isn't necessary at all. Just use the previous `TOPICS` variable for iteration and access the `name` attribute. e.g. `TOPICS[0].name`.